Sources/SuperwallKit/Paywall Presentation/Internal Presentation/PaywallPresentationFailureReason.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  File.swift
+//  
+//
+//  Created by Yusuf Tör on 23/01/2023.
+//
+
+import Foundation
+
+/// The reason to why the paywall couldn't present.
+public enum PaywallPresentationFailureReason {
```

Comment:
I don't think this needs to be `public`

Reply:
It does, it's in the SuperwallEvent `paywallPresentationFail(reason:)`